test/buf-ring-nommap.c

```diff
@@ -46,8 +46,10 @@ int main(int argc, char *argv[])
 	p.flags = IORING_SETUP_NO_MMAP;
 	ret = io_uring_queue_init_mem(1, &ring, &p, ring_mem, 16384);
 	if (ret < 0) {
-		if (ret == -EINVAL || ret == -ENOMEM)
+		if (ret == -EINVAL || ret == -ENOMEM) {
+			free(ring_mem);
```

Comment:
All the test case changes should be separately done before the sanitize commit. I'll ignore the rest of them in here since I've noted it now :-)

Reply:
Just to make sure I understand it correctly: I should create a commit per changed test and chronologically, that should be before the sanitize commit?

Reply:
You can do one commit fixing up these things in all the tests, the main point is that it should not be bundled with adding sanitization, as they are entirely unrelated to that.